src/synapse/workload.py

```diff
@@ -332,6 +332,27 @@ def enable_forgotten_room_retention(current_yaml: dict) -> None:
     current_yaml["forgotten_room_retention_period"] = "28d"
 
 
+def enable_media_retention(current_yaml: dict) -> None:
+    """Change the Synapse configuration to enable media retention.
+
+    Args:
+        current_yaml: current configuration.
+    """
+    current_yaml["media_retention"] = {
+        "remote_media_lifetime": "14d",
+        "local_media_lifetime": "28d",
+    }
+
+
+def delete_stale_devices(current_yaml: dict) -> None:
```

Comment:
Shouldn't it be named `enable_stale_devices_deletion` ?

Reply:
_stares intensely_.........fine.